logsv0/logs_v0_dataprime.go

```diff
@@ -360,22 +369,6 @@ func UnmarshalApisDataprimeV1BytesScannedLimitWarning(m map[string]json.RawMessa
 	return
 }
 
-// ApisDataprimeV1CompileWarning : ApisDataprimeV1CompileWarning struct
-type ApisDataprimeV1CompileWarning struct {
-	WarningMessage *string `json:"warning_message,omitempty"`
-}
-
-// UnmarshalApisDataprimeV1CompileWarning unmarshals an instance of ApisDataprimeV1CompileWarning from the specified map of raw messages.
-func UnmarshalApisDataprimeV1CompileWarning(m map[string]json.RawMessage, result interface{}) (err error) {
-	obj := new(ApisDataprimeV1CompileWarning)
-	err = core.UnmarshalPrimitive(m, "warning_message", &obj.WarningMessage)
-	if err != nil {
-		return
-	}
-	reflect.ValueOf(result).Elem().Set(reflect.ValueOf(obj))
-	return
```

Comment:
Why these are removed?

Reply:
because background query api also uses that and those apis (/status), (/submit)(/cancel) are generated